sites/cnfpt/src/backend/base/tests/test_cache.py

```diff
@@ -0,0 +1,83 @@
+"""Test funmooc cache plugin"""
+
+from unittest import mock
+
+from django.core.cache.backends.dummy import DummyCache
+from django.test import TestCase, override_settings
+
+from django_redis.cache import RedisCache
+
+from base.cache import RedisCacheWithFallback
+
+
+def get_client(**kwargs):
+    return RedisCacheWithFallback(None, kwargs)
+
+
+def get_invalidation_fallback_needed(client):
+    return client._fallback_cache.get("FALLBACK_CACHE__INVALIDATION_REQUIRED")
+
+
+class RedisCacheWithFallbackTestCase(TestCase):
+    """
+    Test suite for the RedisCacheWithFallback
+
+    Credits:
+    - https://github.com/Kub-AT/django-cache-fallback
+    """
```

Comment:
It might be worth testing the throttle function.  Can be done by mocking the `now` method? https://github.com/openfun/richie/blob/85e0a6aa3a4bff71cad8520089605009af359d5e/tests/apps/search/test_index_manager.py#L139